ontology/docs/systems_and_equipment.md

```diff
@@ -0,0 +1,67 @@
+# Modeling Systems and Equipment
+
+This project aims to provide strong guidance in modeling; this is to ensure that all models retain certain commonalities, even while being constructed by disparate modelers with different system contexts. This document is intended to outline some common system types and how they may be modeled effectively and consistently.
```

Comment:
Reworded:

Consistency in modeling is a prerequisite to scalable smart building applications.  As a result, the published models in the DigitalBuildings project are opinionated about how different types of systems are modeled. This document covers many common system types, and describes how to model them consistently with the rest of the ontology.